src/appmixer/supabaseManagement/lib.generated.js

```
@@ -0,0 +1,93 @@
+            const fileName = `${context.config.outputFilePrefix || DEFAULT_PREFIX}-${componentName}.csv`;
+            const savedFile = await context.saveFileStream(pathModule.normalize(fileName), buffer);
+
+            await context.log({ step: 'File was saved', fileName, fileId: savedFile.fileId });
+            await context.sendJson({ fileId: savedFile.fileId }, outputPortName);
+        } else {
+            throw new context.CancelError('Unsupported outputType ' + outputType);
+        }
+    },
+
+    getProperty(obj, path) {
+        return path.split('.').reduce((acc, part) => acc && acc[part], obj);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)